README.rst

```diff
@@ -1,7 +1,11 @@
 BrowserID Page Object Model
 ===========================
 
-Selenium compatible page object model for
+BIDPOM is no longer in active development, and support has been discontinued.
+
+Documentation may be found `here <https://github.com/mozilla/bidpom/wiki>`_.
```

Comment:
I think we should keep the resources section, but maybe just remove the issues link. The others are still potentially useful, particularly when viewing the package on PyPI. I also think we might want to add the notice to the initial wiki page, in case that's used as an entry point.